IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -7,22 +7,22 @@
 # 
 # Extract this information based on ID and save details.
 
-# In[8]:
+# In[20]:
```

Comment:
I think you added the sequentially executed `.ipynb` file. Can you also make sure to re-`nbconvert` and add this updated file as well?

Reply:
I'll address this comment last in case I end up running chunks again which will mess up the execution order.

Reply:
@gwaybio `.py` scripts were generated from `.ipynb` files incorporating changes from this pull request. Both files were also run through black formatting.

Added in 85e51fc1c47cb4f8431d5723ebbd43e7bebf577c.